configure.ac

```diff
@@ -752,6 +754,26 @@ AS_IF([test x$with_lustre != xno],[
        MOTR_LINUX_SENDFILE
 ])
 
+#
+# Checking libfabric library----------------------------------------------- {{{1
+#
+
+AC_ARG_WITH([libfab],
+        [AS_HELP_STRING([--with-libfab=path],
+                        [set path to libfabric library, default is /usr/lib64/])],
+        [FAB_LIBS="-L$with_libfab -lfabric"],
+        [FAB_LIBS="-lfabric"]
```

Comment:
Should this be:
```
 [FAB_LIBS=""]
```


Otherwise, if no "--with-libfab=xxx" is provided (and libfabric is not install), Motr building will fail with:
```
  CC       xcode/ff2c/xcode_ff2c_m0ff2c-ff2c.o
  CC       xcode/ff2c/xcode_ff2c_libmotr_xcode_ff2c_la-lex.lo
  CC       xcode/ff2c/xcode_ff2c_libmotr_xcode_ff2c_la-parser.lo
  CC       xcode/ff2c/xcode_ff2c_libmotr_xcode_ff2c_la-sem.lo
  CC       xcode/ff2c/xcode_ff2c_libmotr_xcode_ff2c_la-gen.lo
  CCLD     xcode/ff2c/libmotr-xcode-ff2c.la
/usr/bin/ld: cannot find -lfabric
collect2: error: ld returned 1 exit status
make[1]: *** [xcode/ff2c/libmotr-xcode-ff2c.la] Error 1
make: *** [pre-all] Error 2
[root@inner cortx-motr]# 
[root@inner cortx-motr]# 
```

Reply:
If no "--with-libfab=xxx" is provided,
the net/libfab/ is still get compiled. So such error will happen:
```
  CC       net/libfab/motr_libmotr_la-libfab.lo
In file included from net/libfab/libfab.c:61:0:
net/libfab/libfab.h:28:24: fatal error: rdma/fi_eq.h: No such file or directory
 #include "rdma/fi_eq.h"
                        ^
compilation terminated.
make[2]: *** [net/libfab/motr_libmotr_la-libfab.lo] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
[root@inner cortx-motr]# 
```

Reply:
So, the question is: When this patch is merged, is the following statement true?
```
We have dependency on libfabric, and libfabric must be installed on system.
Otherwise, the Motr will not compile.
```

Reply:
Yes, 
libfabric must be installed on system, we have added installation step as a part of _./scripts/install-build-deps_

Reply:
> Should this be:
> 
> ```
>  [FAB_LIBS=""]
> ```
> 
> Otherwise, if no "--with-libfab=xxx" is provided (and libfabric is not install), Motr building will fail with:
> 
> ```
>   CC       xcode/ff2c/xcode_ff2c_m0ff2c-ff2c.o
>   CC       xcode/ff2c/xcode_ff2c_libmotr_xcode_ff2c_la-lex.lo
>   CC       xcode/ff2c/xcode_ff2c_libmotr_xcode_ff2c_la-parser.lo
>   CC       xcode/ff2c/xcode_ff2c_libmotr_xcode_ff2c_la-sem.lo
>   CC       xcode/ff2c/xcode_ff2c_libmotr_xcode_ff2c_la-gen.lo
>   CCLD     xcode/ff2c/libmotr-xcode-ff2c.la
> /usr/bin/ld: cannot find -lfabric
> collect2: error: ld returned 1 exit status
> make[1]: *** [xcode/ff2c/libmotr-xcode-ff2c.la] Error 1
> make: *** [pre-all] Error 2
> [root@inner cortx-motr]# 
> [root@inner cortx-motr]# 
> ```

if --with-libfab is not provided then we would be using default libfabric library